src/gdcmodels/esmodels/gdc_from_graph/case/vestigial.yaml

```diff
@@ -61,6 +61,12 @@ dictionary_item_added:
     type: double
   root['properties']['exposures']['properties']['height']:
     type: double
+  root['properties']['exposures']['properties']['marijuana_use_per_week']:
```

Comment:
Are we going manually add vestigial entries as part of the dictionary release?

Reply:
If by manually, you mean we'll need to run the sync script w/ the updated models. Then that is correct. But we should NOT be manually editing these files. That should be left to the sync script.

Reply:
Great! Because I wanted no part of manually editing anything!